examples/text-layer/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -24,7 +24,8 @@ export default class MultiIconLayer extends IconLayer {
   getShaders() {
     const multiIconVs = this.enable64bitSupport(this.props) ? vs64 : vs;
     return Object.assign({}, super.getShaders(), {
-      vs: multiIconVs
+      vs: multiIconVs,
+      modules: ['picking']
```

Comment:
Since this layer subclasses the icon layer, `super.getShaders()` should have specified `modules: ['picking']`. So we can remove it here.

Reply:
Thanks for catching it, I did not add in the beginning, but seems I added once I was going through all changes and saw this one was missing, will remove it.